<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cSettingsBrowser</class>
 <widget class="QDialog" name="cSettingsBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
     <item>
      <widget class="QLineEdit" name="lineEdit_folder"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_parent_folder">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="go-previous" resource="icons.qrc">
         <normaloff>:/navigation/icons/go-previous.png</normaloff>:/navigation/icons/go-previous.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_select_folder">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="folder" resource="icons.qrc">
         <normaloff>:/system/icons/folder.svg</normaloff>:/system/icons/folder.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxOpenCLMode">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>Don't use OpenCL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Use OpenCL for MC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Use OpenCL for MC, HQ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Use OpenCL for all</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Use OpenCL for all, HQ</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QComboBox" name="comboBox_filterBy">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>Filter by Name</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Filter by Fractal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Filter by Effect</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filter:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_filterText"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1215</width>
        <height>760</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_load">
       <property name="text">
        <string>Load settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
